python/pip_install/tools/dependency_resolver/dependency_resolver.py

```diff
@@ -88,6 +88,10 @@ def _select_golden_requirements_file(
     requirements_in_path = Path(requirements_in)
     resolved_requirements_in = str(requirements_in_path.resolve())
 
+    requirements_file = _select_golden_requirements_file(
+        requirements_txt, requirements_linux, requirements_darwin, requirements_windows
```

Comment:
nit: I know that this function was created before this PR, but it would be great if we used keyword arguments here for readability and correctness.

It would also help if we return `pathlib.Path` type instance.

Reply:
I at least added the keyword arguments but will skip returning pathlib.Path for now.

Reply:
LGTM